src/components/Header/index.js

```diff
@@ -20,6 +20,7 @@ const Header = () => (
       <MenuLink label='Harm' url='/harm'></MenuLink>
       <MenuLink label='Consequences' url='/consequences'></MenuLink>
       <MenuLink label='Alternatives' url='/alternatives'></MenuLink>
+      <MenuLink label='Contact' url='/contact'></MenuLink>
```

Comment:
We were talking earlier about having some logic that's passed in as to whether the Contact page is created or not for the specific site. Maybe we should either do that next, or make a note to do that when we are handling deployment.